sklearn/covariance/_elliptic_envelope.py

```diff
@@ -67,6 +67,7 @@ class EllipticEnvelope(OutlierMixin, MinCovDet):
         The offset depends on the contamination parameter and is defined in
         such a way we obtain the expected number of outliers (samples with
         decision function < 0) in training.
+        .. versionadded:: v0.20.0
```

Comment:
```suggestion
        .. versionadded:: 0.20
```

I think this is how it's done in other places.

Please also fix the other ones.